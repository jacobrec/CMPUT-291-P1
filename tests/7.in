1
jacob
password
1
1
o
p
l
m
;
0
n
1
16
y
YYYY-MM-DD
2016-00-00
2016-01-00
2016-13-31
2016-12-32

Going too fast
-10
1
0
16
y
2016-08-08
Being a jerk
1000
3
2
