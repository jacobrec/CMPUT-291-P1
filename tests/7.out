Please sign in to use App:
1) login
2) exit
Username: Password: Enter a command:
1) issue a ticket
2) find a car owner
3) signout
Registration Number: │  First Name   │   Last Name   │     Make      │     Model     │     Color     │     Year      │
╞═══════════════╪═══════════════╪═══════════════╪═══════════════╪═══════════════╪═══════════════╡
│     Sage      │      Fox      │   Chevrolet   │    Camaro     │      red      │     1969      │
Issue Ticket: y/n
Please type y/n
Please type y/n
Please type y/n
Please type y/n
Please type y/n
Please type y/n
Enter a command:
1) issue a ticket
2) find a car owner
3) signout
Registration Number: │  First Name   │   Last Name   │     Make      │     Model     │     Color     │     Year      │
╞═══════════════╪═══════════════╪═══════════════╪═══════════════╪═══════════════╪═══════════════╡
│    Speedy     │  McSpeedFace  │     Zoom      │     Fast      │      Red      │     2017      │
Issue Ticket: y/n
Violation Date: Invalid input for date, enter YYYY-MM-DD or leave blank to get todays date
Violation Date: Invalid input for date, enter YYYY-MM-DD or leave blank to get todays date
Violation Date: Invalid input for date, enter YYYY-MM-DD or leave blank to get todays date
Violation Date: Invalid input for date, enter YYYY-MM-DD or leave blank to get todays date
Violation Date: Invalid input for date, enter YYYY-MM-DD or leave blank to get todays date
Violation Date: Violation Text: Fine Amount: Enter a command:
1) issue a ticket
2) find a car owner
3) signout
Registration Number: Registration not found, try again
Registration Number: │  First Name   │   Last Name   │     Make      │     Model     │     Color     │     Year      │
╞═══════════════╪═══════════════╪═══════════════╪═══════════════╪═══════════════╪═══════════════╡
│    Speedy     │  McSpeedFace  │     Zoom      │     Fast      │      Red      │     2017      │
Issue Ticket: y/n
Violation Date: Violation Text: Fine Amount: Enter a command:
1) issue a ticket
2) find a car owner
3) signout
Please sign in to use App:
1) login
2) exit
